thumbor_plugins/__init__.py

```diff
@@ -5,4 +5,4 @@
 # http://www.opensource.org/licenses/mit-license
 # Copyright (c) 2011 globo.com timehome@corp.globo.com
 
-__version__ = "0.2.4"
+__version__ = "1.0.0"
```

Comment:
Maybe wait until we finish moving optimizers from old_optimizers to optimizers before 1.0.0?

Reply:
This version doesn´t matter anymore, now all the plugins are released with thumbor-plugins-<optimizer>.